### PR TITLE
update: readme files

### PR DESCRIPTION
**Pull request by AlejandraPedroza**
Updating readme files of browser-example, compose-imageviewer, compose-jetsnack, nodejs-example, wasi-example, and main readme.

### CODE REVIEW

**Review by JSMonk — APPROVED** [review]

File: compose-jetsnack/README.md [hunk 1]
```diff
@@ -1,80 +1,121 @@
...
+Compose Multiplatform for web enables experimenting with sharing your mobile or desktop UIs with the web.
+
+Compose Multiplatform for web is based on [Kotlin/Wasm](https://kotl.in/wasm), the newest target for Kotlin Multiplatform projects.
+It allows you to run your code in the browser with all WebAssembly's benefits, such as high and predictable application performance.
 
-> **Note**
-> Web support is Experimental and may be changed at any time. Use it only for evaluation purposes.
-> We would appreciate your feedback on it in the public Slack channel [#compose-web](https://slack-chats.kotlinlang.org/c/compose-web).
-> If you face any issues, please report them on [GitHub](https://github.com/JetBrains/compose-multiplatform/issues).
+In the next sections, you can find information to try out the Jetsnack application built with Compose Multiplatform for web and Kotlin/Wasm.
```
Comment (daniCsorbaJB) [inline compose-jetsnack/README.md:42]:
```suggestion
Follow the instructions in the sections below to try out this Jetsnack app built with Compose Multiplatform for web and Kotlin/Wasm.
```

File: compose-jetsnack/README.md [hunk 2]
```diff
@@ -1,80 +1,121 @@
...
 
-Compose for Web is based on [Kotlin/Wasm](https://kotl.in/wasm), the newest target for Kotlin Multiplatform projects.
-It allows you to run your code in the browser with all the benefits that WebAssembly has to offer, such as good and predictable performance for your applications.
+## Set up the environment
 
-## Setup environment
+Before you start, prepare the IDE and browser you require to run the application.
```
Comment (daniCsorbaJB) [inline compose-jetsnack/README.md:46]:
```suggestion
Before starting, ensure you have the necessary IDE and browser setup to run the app.
```

File: compose-jetsnack/README.md [hunk 3]
```diff
@@ -1,80 +1,121 @@
...
+> For more information about the browser versions, see the [Troubleshooting documentation](https://kotl.in/wasm_help/).
 
 ## Build and run
 
-Check out the repository, navigate to the project folder, and use the following commands:
+To build and run the Jetsnack application with Compose Multiplatform for web and Kotlin/Wasm:
+
+1. In IntelliJ IDEA, open the repository.
+2. Navigate to the `compose-jetsnack` project folder.
+3. Run the application by using the commands explained in the following sections.
```
Comment (daniCsorbaJB) [inline compose-jetsnack/README.md:83]: same as above - consider combining the two sections into an unordered list and updating step 3

File: compose-jetsnack/README.md [hunk 4]
```diff
@@ -1,80 +1,121 @@
...
+
+`./gradlew :web:wasmJsRun`
+
+Once the application starts, open the following URL in your browser: 
+
+`http://localhost:8080`
 
-### Run Web version via Gradle
+> **Note:**
+> The port number can vary. If the 8080 port is unavailable, you can find the corresponding port number printed in the console
```
Comment (daniCsorbaJB) [inline compose-jetsnack/README.md:96]:
```suggestion
> The port number can vary. If port 8080 is unavailable, you can find the corresponding port number printed in the console
```

File: nodejs-example/README.md [hunk 5]
```diff
@@ -1,23 +1,40 @@
-# Kotlin/Wasm Node.js Example
+# Kotlin/Wasm Node.js example
 
-A simple app printing "Hello World" in the Node.js.
+This example shows a simple application printing "Hello World" in Node.js.
```
Comment (daniCsorbaJB) [inline nodejs-example/README.md:3]:
```suggestion
This example showcases a simple app printing "Hello World" in Node.js.
```

File: nodejs-example/README.md [hunk 6]
```diff
@@ -1,23 +1,40 @@
...
 
-Check out the repo, navigate to the project folder, and use the following command:
+To build and run the application:
 
-### Run via Gradle
+1. In IntelliJ IDEA, open the repository.
+2. Navigate to the `nodejs-example` project folder.
+3. Run the application by using the following Gradle command:
 
 `./gradlew wasmJsNodeRun`
```
Comment (daniCsorbaJB) [inline nodejs-example/README.md:29]: I think we could add this command in step 3

File: wasi-example/README.md [hunk 7]
```diff
@@ -1,37 +1,66 @@
-# Kotlin/Wasm WASI Example
+# Kotlin/Wasm WASI example
 
-A simple app using WASI API in the Node.js and Deno.
+This example shows a simple application using WASI API in Node.js and Deno.
```
Comment (daniCsorbaJB) [inline wasi-example/README.md:3]:
```suggestion
This example showcases a simple app using WASI API in Node.js and Deno.
```

File: wasi-example/README.md [hunk 8]
```diff
@@ -1,37 +1,66 @@
...
+It supports Kotlin/Wasm out of the box.
 
 ## Build and run
 
-Check out the repo, navigate to the project folder, and use the following command:
+To build and run the application:
+
+1. In IntelliJ IDEA, open the repository.
+2. Navigate to the `wasi-example` project folder.
+3. Run the application by using the commands explained in the following sections.
```
Comment (daniCsorbaJB) [inline wasi-example/README.md:27]: Consider turning this into an unordered list instead of sections as well.

File: wasi-example/README.md [hunk 9]
```diff
@@ -1,37 +1,66 @@
...
 
 ### Run tests via Gradle with Deno
 
+To run tests with Deno, type the following Gradle command in the terminal:
+
 `./gradlew wasmWasiDenoTest`
 
-(for Windows platform deno.exe expected to be installed)
+> **Note:**
+> For Windows platform, `deno.exe` is expected to be installed. For more information,
```
Comment (daniCsorbaJB) [inline wasi-example/README.md:54]:
```suggestion
> For Windows platform, ensure `deno.exe` is installed. For more information, 
```

File: browser-example/README.md [hunk 10]
```diff
@@ -1,40 +1,67 @@
...
+  Works by default.
 
-We recommend using [IntelliJ IDEA 2023.1 or newer](https://www.jetbrains.com/idea/) to work with the project.
+**Safari/WebKit**
+
+Wasm GC support is currently under
+[active development](https://bugs.webkit.org/show_bug.cgi?id=247394).
+
+> **Note:**
+> For more information about the browser versions, see the [Troubleshooting documentation](https://kotl.in/wasm_help/).
```
[the remark this answers is not in the document]
Comment (AlejandraPedroza) [reply browser-example/README.md:44]: I did a quick research in our docs as well as in our style guide and I found "For more information about" more frequent than "detail on". I'll leave as it is to keep consistency. WDYT?

File: compose-imageviewer/README.md [hunk 11]
```diff
@@ -1,61 +1,101 @@
...
 
 ## Build and run
 
-Check out the repo, navigate to the project folder, and use the following commands:
+To build and run the application:
 
-### Run Web version via Gradle
+1. In IntelliJ IDEA, open the repository.
+2. Navigate to the `compose-imageviewer` project folder.
+3. Run the application by using the commands explained in the following sections.
```
[the remark this answers is not in the document]
Comment (AlejandraPedroza) [reply compose-imageviewer/README.md:62]: Great suggestion. I applied to this and the rest of the examples that have multiple commands.

File: nodejs-example/README.md (hunk 6 above)
Comment (AlejandraPedroza) [reply nodejs-example/README.md:29]: I tried it, but it is not that visible and hard to scan, I'll leave it as it is in a separate line.

File: browser-example/README.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (daniCsorbaJB) [reply browser-example/README.md:44]: sounds good to me!

**Pull request merged**